arkane/statmech.py

```diff
@@ -1103,6 +1103,7 @@ def assign_frequency_scale_factor(freq_level):
                  'ccsd(t)-f12/aug-cc-pvtz': 0.998,  # [3], taken as CCSD(T)-F12a/cc-pVTZ-F12
                  'ccsd(t)-f12/aug-cc-pvqz': 0.998,  # [3], taken as 'CCSD(T)-F12b/VQZF12//CCSD(T)-F12a/TZF'
                  }
+    freq_level.replace('**', '(d,p)')
```

Comment:
I don't think I understand, why do we want to use BAC from one basis set and freq scaling factor from a different basis?